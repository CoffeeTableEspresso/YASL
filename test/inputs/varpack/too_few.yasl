




const a, const b, const c = 1, 2

echo a
echo b
echo c
