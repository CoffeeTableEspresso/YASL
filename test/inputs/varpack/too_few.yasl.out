1
2
undef
